include/factory_data.h

```diff
@@ -28,10 +28,12 @@
  * Do not access fields directly. Use API functions only. */
 typedef struct {
```

Comment:
Good point, I'm hoping this can be finalized tomorrow to lock in whatever is actually being used for DVT/PVT and beyond.